doc/ros_api.md

```diff
@@ -133,6 +133,21 @@ Check the relevant fields of the `RobotStatus` messages to determine overall con
 
 The `reset_error` service can be used to attempt to reset errors and alarms
 
+### start_rt_mode
+
+Type: [motoros2_interfaces/srv/StartRtMode](https://github.com/yaskawa-global/motoros2_interfaces/srv/StartRtMode.srv)
+
+Attempts to enable servo drives, activate the real-time UDP server, and set the job-cycle mode to allow execution of INIT_ROS.
```

Comment:
suggestion: link to the documentation of *the real-time UDP server* here.

That link is also two lines down, but it's a bit confusing to refer to *the [..] server* here without immediately linking to it.